Dockerfile

```diff
@@ -13,12 +14,11 @@ RUN apt-get -y update
 RUN apt-get -y install scons
 RUN m 3.6
 RUN m tools stable
-# Load balance at least 2 copies
 RUN npm install -g pm2
-COPY . /app
-ENV APOS_MINIFY=1
-ENV NODE_ENV=production
+COPY . ./
 RUN ./scripts/docker-build-assets-with-temporary-mongod
 RUN m rm 3.6
 EXPOSE 3000
-CMD ./scripts/docker-start
+ENTRYPOINT [ "./scripts/docker-wait-for-port", "mongo:27017", "--" ]
```

Comment:
I think the -- is superfluous now that we're not using it to add pm2-runtime on the same line.

Reply:
Seems like we do need it. I tried it without the -- but it doesn't work. The ENTRYPOINT is always prepended to the CMD so they do run on the same line.